modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/idp/mgt/IdentityProviderMgtServiceTestCase.java

```diff
@@ -72,6 +77,11 @@ public void testInit() throws Exception {
     @AfterClass(alwaysRun = true)
     public void atEnd() throws Exception {
 
+        // Delete the Idp tested.
+        deleteIDP();
+        IdentityProvider idp = idpMgtServiceClient.getIdPByName(testIdpNameSearch);
+        Assert.assertNull(idp, "Deleting idp failed");
```

Comment:
not required to assert in atEnd method. only calling idpMgtServiceClient.deleteIdP(testIdpNameSearch) is sufficient here